tendrl/node_agent/persistence/disk.py

```diff
@@ -0,0 +1,31 @@
+from tendrl.common.etcdobj.etcdobj import EtcdObj
+from tendrl.common.etcdobj import fields
+
+
+class Disk(EtcdObj):
+    """A table of the Disk, lazily updated
+
+    """
+    __name__ = 'nodes/%s/Disk/%s'
```

Comment:
URl should be like nodes/<nodeid>/disks/<diskid> or nodes/<nodeid>/devices/<deviceid>